### PR TITLE
INT-7341 new red hat registries

### PR DESCRIPTION
**Pull request by jflinchbaugh**
switch to new red hat registry and credentials for building the red hat certified image.

JIRA: https://issues.sonatype.org/browse/INT-7341
Build: https://jenkins.ci.sonatype.dev/job/integrations/job/cloud/job/Red%20Hat/job/release-test/1/

### CODE REVIEW

File: Jenkinsfile.rh [hunk 1]
```diff
@@ -24,18 +24,19 @@ node('ubuntu-zion') {
       sh 'docker system prune -a -f'
       sh '''
         wget -q -O preflight \
-          https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/1.4.0/preflight-linux-amd64
+          https://github.com/redhat-openshift-ecosystem/openshift-preflight/releases/download/1.4.1/preflight-linux-amd64
```
Comment (jflinchbaugh) [inline Jenkinsfile.rh:27]: latest preflight tool.

File: build_red_hat_image.sh [hunk 2]
```diff
@@ -21,34 +21,33 @@
...
 
 set -x # log commands as they execute
 set -e # stop execution on the first failed command
 
-IMAGE=nexus-repository-manager
 DOCKERFILE=Dockerfile.rh.ubi
 
 # from config/scanning page at red hat
-PROJECT_ID=p725060002a684d7b4fb1475f224f6c26bc3b23998
-# from url of project at red hat
```
Comment (jflinchbaugh) [inline build_red_hat_image.sh:35]: image and project_id are no longer needed.

File: build_red_hat_image.sh [hunk 3]
```diff
@@ -21,34 +21,33 @@
...
 CERT_PROJECT_ID=5e61d90a38776799eb517bd2
 
+REPOSITORY="quay.io"
+IMAGE_TAG="${REPOSITORY}/redhat-isv-containers/${CERT_PROJECT_ID}:${VERSION}"
```
Comment (jflinchbaugh) [inline build_red_hat_image.sh:37]: extracted IMAGE_TAG, so I can stop repeatedly building it.

**Review by amoreno-sonatype — APPROVED** [review]
+1

**Pull request merged**